versions.props

```diff
@@ -1,17 +1,12 @@
 com.fasterxml.jackson.core:jackson-databind = 2.18.3
-org.apache.commons:commons-lang3 = 3.17.0
-com.google.guava:guava = 33.4.0-jre
-com.palantir.baseline:gradle-baseline-java = 6.8.0
-
-# test dependencies
```

Comment:
What led to the refactor of the dependencies here? I guess not all of these were used only in tests?